Description: use headers in /usr/include/libdrm
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 xserver-xorg-video-imx-viv (1-1) unstable; urgency=low
 .
   * Initial release
   * xserver-xorg-video-imx-viv-3.10.17-1.0.1
Author: unknown <unknown>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- xserver-xorg-video-imx-viv-1.orig/EXA/src/makefile.linux
+++ xserver-xorg-video-imx-viv-1/EXA/src/makefile.linux
@@ -54,7 +54,7 @@ endif
 LOCAL_CFLAGS += -fvisibility=hidden -fPIC -DPIC -DFREESCALE_EXTENSION
 LOCAL_CFLAGS += -I. -I.. -I$(sysroot)/usr/include -I$(sysroot)/usr/include/xorg -I$(sysroot)/usr/include/$(PIXMAN)
 LOCAL_CFLAGS += -I./vivante_gal -I./vivante_fbdev -I./vivante_util -I./vivante_exa -I./vivante_extension
-LOCAL_CFLAGS += -I$(sysroot)/usr/include/drm
+LOCAL_CFLAGS += -I$(sysroot)/usr/include/libdrm
 
 LOCAL_LFLAGS += -fPIC
 LOCAL_LFLAGS += -L$(sysroot)/usr/lib -lm -ldl -lGAL -Wl,-soname -Wl,vivante_drv.so
--- xserver-xorg-video-imx-viv-1.orig/FslExt/src/makefile.linux
+++ xserver-xorg-video-imx-viv-1/FslExt/src/makefile.linux
@@ -27,7 +27,7 @@ OBJS := fsl_pixmap_ext.o \
 # macro FREESCALE_EXTENSION is reserved for future use
 LOCAL_CFLAGS += -fPIC 
 LOCAL_CFLAGS += -I. -I.. -I$(sysroot)/usr/include -I$(sysroot)/usr/include/xorg
-LOCAL_CFLAGS += -I$(sysroot)/usr/include/drm
+LOCAL_CFLAGS += -I$(sysroot)/usr/include/libdrm
 
 LOCAL_LFLAGS += -fPIC
 LOCAL_LFLAGS += -L$(sysroot)/usr/lib -lGAL -lXext -lX11 -lc -Wl,-soname -Wl,libfsl_x11_ext.so
